randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -60,15 +60,15 @@ Extra - minimap_data: [{'x': 53, 'y': 22}, {'x': 53, 'y': 23}, {'x': 53, 'y': 24
   > Bottom Pit
       High Mid-Air Morph Tunnel Climb and Low Mid-Air Morph Tunnel Climb
 
-> Horizontal Dock to EMP Sentry Cavern; Heals? False
+> Horizontal Dock to EMP Sentry Cavern (Top); Heals? False
   * Layers: default
-  * 4-Tiles High Dock to EMP Sentry Cavern/Horizontal Dock to Pincherfly Parlor
+  * 4-Tiles High Dock to EMP Sentry Cavern/Horizontal Dock to Pincherfly Parlor (Top)
   > Top
       Super Missiles
 
-> Dock to EMP Sentry Cavern; Heals? False
+> Horizontal Dock to EMP Sentry Cavern (Bottom); Heals? False
```

Comment:
The bottom one should be 5 tiles high, not 4
![grafik](https://github.com/randovania/randovania/assets/38186597/22e4a7a4-9b59-41f4-a3e4-249ea71d3e68)